flix-core/src/main/java/com/epam/mentoring/flixcore/controller/LoginController.java

```diff
@@ -37,18 +35,18 @@ public ModelAndView registration() {
         return modelAndView;
     }
 
-    @RequestMapping(value = "/registration", method = RequestMethod.POST)
+    @PostMapping(value = "/registration")
     public ModelAndView createNewUser(@Valid User user, BindingResult bindingResult) {
```

Comment:
small hint here - BindingResult MUST be just after @Valid bean - you did it correct, but maybe accidentally :)
Keep this in mind coz if its not next it just wont work and wont tell you why. Very nasty bug.